vast.go

```diff
@@ -345,15 +349,15 @@ type NonLinear struct {
 	// The apiFramework defines the method to use for communication with the nonlinear element.
 	APIFramework string `xml:"apiFramework,attr,omitempty"`
 	// URLs to ping when user clicks on the the non-linear ad.
-	NonLinearClickTracking []string `xml:",omitempty"`
+	NonLinearClickTracking []CDATA `xml:",omitempty"`
```

Comment:
did you try `NonLinearClickTracking []string `xml:",cdata,omitempty`? Try adding a test with it withCDATA and without. If it works, please change all of them and remove the `CDATA` struct